openfisca_us/variables/finance/benefit/ccdf.py

```diff
@@ -46,6 +46,9 @@ class is_ccdf_age_eligible(Variable):
     definition_period = YEAR
     label = u"Age eligibility for CCDF"
 
+    def formula(person, period):
+        return person("age", period) < 13
```

Comment:
Actually could you make this a parameter?

Reply:
Yes, I'll do that.